### ISSUE TITLE
[ACCESSIBILITY] Intro video does not have Captions or media alternative

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1570]
**Describe the accessibility issue**
[Captions](https://www.w3.org/TR/WCAG21/#dfn-captions) are provided for all [prerecorded](https://www.w3.org/TR/WCAG21/#dfn-prerecorded) [audio](https://www.w3.org/TR/WCAG21/#dfn-audio) content in [synchronized media](https://www.w3.org/TR/WCAG21/#dfn-synchronized-media), except when the media is a [media alternative for text](https://www.w3.org/TR/WCAG21/#dfn-media-alternative-for-text) and is clearly labeled as such.

[Success Criterion 1.2.2 Captions (Prerecorded)](https://www.w3.org/TR/WCAG21/#captions-prerecorded)
(Level A)
An [alternative for time-based media](https://www.w3.org/TR/WCAG21/#dfn-alternative-for-time-based-media) or [audio description](https://www.w3.org/TR/WCAG21/#dfn-audio-descriptions) of the [prerecorded](https://www.w3.org/TR/WCAG21/#dfn-prerecorded) [video](https://www.w3.org/TR/WCAG21/#dfn-video) content is provided for [synchronized media](https://www.w3.org/TR/WCAG21/#dfn-synchronized-media), except when the media is a [media alternative for text](https://www.w3.org/TR/WCAG21/#dfn-media-alternative-for-text) and is clearly labeled as such.
[Success Criterion 1.2.3 Audio Description or Media Alternative (Prerecorded)](https://www.w3.org/TR/WCAG21/#captions-prerecorded)

**To Reproduce
Steps to reproduce the behavior:**

1. Go to freshly installed version of app (my instance was 3.3.15 apk)
2. Go through app orientation screens
3. Upon arriving at third screen video plays with no captions or media alternative offered. 

Expected behavior
Upon arriving at third screen video plays with no captions or media alternative offered. 

**Screenshots**
![Screenshot_20220420-143220](https://user-images.githubusercontent.com/82892873/164309870-48321cb6-2706-439e-ac28-e228bd8c034d.png)

**Smartphone (please complete the following information):**
Pixel 4a- Android 12

Additional context

**Comment by acovher2** [comment]
Verified as fixed in 4.0.49 Android apk. 
Closing. 
Thank you,
Amanda
